mantle/cmd/kola/resources/iscsi_butane_setup.yaml

```diff
@@ -29,14 +29,15 @@ storage:
             [Install]
             # Start by default on boot
             WantedBy=multi-user.target
-            ExecStop=podman stop --time 1 target
+            [Service]
+            ExecStop=/usr/bin/podman stop --time 1 targetd
```

Comment:
Hmm, we shouldn't need this (though does it work?). Let's see in https://github.com/coreos/fedora-coreos-config/pull/2816 if we can figure this out without resorting to it.